csm-extensions/services/dev-mysql/Dockerfile-setup

```diff
@@ -1,17 +1,20 @@
-FROM cf-usb-sidecar-buildbase:latest
-
-COPY chart/scf-connector.sh /usr/local/bin/
-COPY chart/authorize_ca.sh  /usr/local/bin/
-WORKDIR /
-
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y unzip
+FROM cf-usb-sidecar-buildbase:latest as builder
 
-RUN wget -O -  'https://cli.run.pivotal.io/stable?release=linux64-binary&source=github' | tar xfz - -C /usr/local/bin cf \
-    && wget -O cf-plugin-usb.zip https://github.com/SUSE/cf-usb-plugin/releases/download/0.0.1/cf-plugin-usb-linux-amd64.zip \
-    && /usr/bin/unzip cf-plugin-usb.zip \
-    && mv cf-plugin-usb /usr/local/bin \
-    && rm cf-plugin-usb.zip
-
-RUN chmod +x /usr/local/bin/cf /usr/local/bin/cf-plugin-usb /usr/local/bin/scf-connector.sh
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y unzip
+RUN mkdir /out
+RUN wget -O cf.tgz 'https://cli.run.pivotal.io/stable?release=linux64-binary'
+RUN tar xzf cf.tgz -C /out cf
+RUN wget -O cf-plugin-usb.zip https://github.com/SUSE/cf-usb-plugin/releases/download/0.0.1/cf-plugin-usb-linux-amd64.zip
+RUN unzip cf-plugin-usb.zip
+RUN mv cf-plugin-usb /out
+COPY chart/scf-connector.sh /out/
+COPY chart/authorize_ca.sh  /out/
+RUN chmod +x /out/cf /out/cf-plugin-usb /out/scf-connector.sh
 
+FROM cf-usb-sidecar-buildbase:latest
+COPY --from=builder /out/ /usr/local/bin/
+WORKDIR /
 ENTRYPOINT /bin/bash /usr/local/bin/scf-connector.sh
+COPY --from=builder /out/ /usr/local/bin/
```

Comment:
This command seems to be a dup of line 17. However with the workdir/entrypoint change it might run ina different context ?

Reply:
No, you're right, that's not supposed to be there.

Reply:
Should I check the other services for similar dups ?